src/batou/utils.py

```diff
@@ -167,7 +167,7 @@ def __init__(self, connect_address, port=None):
             self.listen_v6 = NetLoc(v6_address, str(port))
 
         if not self.listen and not self.listen_v6:
-            raise socket.gaierror("No v4 or v6 address for %s" % connect)
+            raise socket.gaierror("No v4 or v6 address for '%s'." % connect)
```

Comment:
Python has a standard for "literal quoting". I'd recommend to switch to %r, or _actually_ this is Python 3, please use "".format().